cirq/testing/random_dag.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Iterable
+
+from cirq import circuits
+from cirq.testing.random_circuit import random_circuit
+
+
+def random_circuit_dag(*args, **kwargs) -> circuits.Circuit:
```

Comment:
This method isn't doing enough to justify itself, and it acts as a documentation barrier by having indirectly-defined mystery arguments. Delete it and inline it where you're using it.

Reply:
Done